client/src/Admin/Settings.js

```diff
@@ -1,12 +1,77 @@
-import React from 'react';
+import React, { useState, useContext, useEffect } from 'react';
+
+import Context from '../Context';
+import ApiService from '../ApiService';
+import FormError from '../Models/FormError';
+import FormCheckbox from '../Components/FormCheckbox';
+
+import './Settings.scss';
 
 function Settings() {
+  const { organization, setOrganization } = useContext(Context);
+  const [organizationInEdit, setOrganizationInEdit] = useState(organization);
+  const [error, setError] = useState();
+  const [showEdit, setShowEdit] = useState(false);
+
+  useEffect(() => {
+    setOrganizationInEdit(organization);
+    setError();
+  }, [showEdit, organization]);
```

Comment:
resetting the `organizationInEdit` when showEdit and organization is toggled such that it's always in sync with `organization` context when we close out of Edit mode without saving and for when we update the organization's info in our context.